packages/daylight/start.go

```diff
@@ -172,6 +172,15 @@ func initRoutes(listenHost string) {
 	httpListener(listenHost, route)
 }
 
+func logBlockchainMode() {
```

Comment:
useless function, used only 1 time

Reply:
The use of this function makes the start() function code more cleaner